SprawdzanieNIPu/ConsoleApplication1/NIP.cs

```diff
@@ -1,72 +1,68 @@
-﻿using System;
+﻿namespace ConsoleApplication1
+ {
+    using System;
 
-namespace ConsoleApplication1
-{
-    class NIP
-    {
-        public string numer;
-
-        public NIP(string nr)
-        {
-            this.numer = nr.Trim();
-        }
-
-        public Boolean dlugoscOK
-        {
-            get
-            {
-                return this.numer.Length == 10;
-            }
-        }
-
-        public int kontrolna
-        {
-            // zwraca sumę kontrolną
-            get
-            {
-                int[] wagi = new int[] {6, 5, 7, 2, 3, 4, 5, 6, 7};
-                double x=0;
-                double liczba=0;
+ /// <summary>
+ /// Klasa służy do sprawdzenia, czy wprowadzony ciąg znaków jest poprawnym numerem NIP i jego sformatowania.
+ /// </summary>
+ internal class NIP
+ {
+   private long numer;
```

Comment:
Nie rozumiem - liczba reprezentująca NIP jest na tyle długa, że potrzebuję Int64, czyli long ([https://msdn.microsoft.com/en-us/library/ctetwysk.aspx](url)

Dlaczego decimal, w czym to pomoże?